### DISCUSSION TITLE
How To Correctly Use `path=` Selectors?

### DISCUSSION THREAD

**Discussion by stevekuznetsov** [community/community#38357] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm running [this](https://cs.github.com/?scopeName=All+repos&scope=&q=%2FParse.*%5C%28.*resourceVersion%5C%29%2F) query and want to exclude forks and copies of `kubernetes/kubernetes` and `kubernetes/apiserver`. Since the `repo=` keyword a) requires the owner prefix and b) doesn't allow fuzzy matching, I can instead filter paths.

However, using `AND NOT path=<anything>` just doesn't seem to work. Similarly, a trivial `AND path=*.go` doesn't work either. Am I misusing the search bar, or is the index broken, or something?
![Screenshot from 2022-11-05 08-21-09](https://user-images.githubusercontent.com/7328370/200125260-60be380b-036d-4df7-8c73-76e8b928f205.png)
![Screenshot from 2022-11-05 08-20-51](https://user-images.githubusercontent.com/7328370/200125261-4c5063c8-0b9e-46da-b0ce-a116513914c0.png)
![Screenshot from 2022-11-05 08-11-32](https://user-images.githubusercontent.com/7328370/200125263-03d84276-01ae-4f8c-b4b9-2472be1c7103.png)
![Screenshot from 2022-11-05 08-10-51](https://user-images.githubusercontent.com/7328370/200125264-98176be8-3c2f-4d8a-b58f-472123b9bf45.png)

**Comment by ossanna16** [comment]
Hi there @stevekuznetsov and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our official introduction thread
To connect with other new users, check out this Discussion to Meet Your GitHub Community Team, further your GitHub knowledge by working through our GitHub Skill 🆙 exercises, and participate in our weekly Fun Friday chats

**Comment by look** [comment, marked as answer]
Hi @stevekuznetsov, maybe we need a query linter for this, but since you are using `path=...`, the search engine is interpreting that as an exact match for that string, including the `path=` part.

To search for paths, use `path:`, for example [`/Parse.*\(.*resourceVersion\)/ AND path:*.go`](https://cs.github.com/?scopeName=All+repos&scope=&q=%2FParse.*%5C%28.*resourceVersion%5C%29%2F+AND+path%3A*.go).

**Reply by stevekuznetsov** [reply]
Ah, thank you!

**Reply by stevekuznetsov** [reply]
No idea how I missed the colon there :facepalm:
